README.md

```diff
@@ -44,15 +38,15 @@ git clone git@github.com:gracekabaghe/budget-app.git
 ```
 cd Budget-app
 ```
-- Install bundles
+Install bundles
 
 ```
 bundle install
 ```
 
-- rails s
 ```
-
+rails s
+```
```

Comment:
- [ ] You are repeating yourself in the testing cases, the tests you wrote for the login seem to be the same as the entity_spec. Please implement the tests for each feature to test if the application is working properly. 

Sample tests:

- should see users' categories
- should show the total amount
- should have a "ADD A NEW CATEGORY" button
- should get an error if either email or password fields are empty
- should get an error if either email or password fields are incorrect
- should redirect to the root page when the user logs in
- should validate user

Please find the other tests and implement them.

![image](https://user-images.githubusercontent.com/55635977/159938226-da7fcbb1-8b5c-4270-9f0d-d35cc41b0241.png)

Reply:
Added more tests